src/content-helper/editor-sidebar/related-posts/hooks.ts

```diff
@@ -36,23 +37,42 @@ interface PostData {
  *
  * @since 3.14.3
  * @since 3.14.4 Implemented checks to reduce risk of invalid data being processed.
+ * @since 3.16.2 Added `isLoading` and `hasResolved` to check if the data is still being fetched or if it
+ * 							 has been resolved.
```

Comment:
Is spacing correct here?

Reply:
Weird, looks fine on my end. I will see if I can adjust it.